TTWWW-548: Content manage footer

mattga11ant opened the pull request:
Content managed the footer based on Sam's comments [here](https://simonsfoundation.atlassian.net/browse/TTWWW-548?focusedCommentId=120401).

Also updated the list styling based on the same comment.

- [x] pull this branch
- [x] apply the migration
- [x] apply the second migration
- [x] log into the backend and click "footers", add a footer.
- [x] Update all fields in the footer to include navigation left and right links and confirm that they show as the design dictates (test HTML in the link title as well).

mattga11ant committed:
TTWWW-548: content managed the footer

mattga11ant committed:
Merge branch 'matt/TTWWW-548-safe' into matt/TTWWW-548

chris-campbell reviewed (changes requested):
The footer looks generally good. However if this is meant as an apples-to-apples comparison with the footer on TT’s main site, it looks like these two areas are missing.

<img width="339" height="363" alt="Screenshot 2025-08-25 at 10 31 15 AM" src="https://github.com/user-attachments/assets/710d0a54-f69d-4791-84ea-71ec65b70777" />

<img width="1463" height="352" alt="Screenshot 2025-08-25 at 10 32 52 AM" src="https://github.com/user-attachments/assets/2bb5a58d-0ecc-471b-8f81-aa4a1380dc48" />

mattga11ant committed:
TTWWW-548: updated footer to include missing line and missing simons logo

mattga11ant committed:
TTWWW-548: made the copywrite year dynamic

mattga11ant commented:
> The footer looks generally good. However if this is meant as an apples-to-apples comparison with the footer on TT’s main site, it looks like these two areas are missing.
> 
> <img alt="Screenshot 2025-08-25 at 10 31 15 AM" width="339" height="363" src="https://private-user-images.githubusercontent.com/16828559/481668123-710d0a54-f69d-4791-84ea-71ec65b70777.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYzMDIxNjQsIm5iZiI6MTc1NjMwMTg2NCwicGF0aCI6Ii8xNjgyODU1OS80ODE2NjgxMjMtNzEwZDBhNTQtZjY5ZC00NzkxLTg0ZWEtNzFlYzY1YjcwNzc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODI3VDEzMzc0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3NWY2YTM0MzU4NmQ4YzMzMjcwZWJiMDMxMWU1OTkyYjFiYzRjZDA3ODRiMTE5MTUxOTcxNWNlOTMxNDlhMTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8noK5Qxq-71WJCP41ERTeHvqss3Sdwnh3ohzbSFA8OU"> <img alt="Screenshot 2025-08-25 at 10 32 52 AM" width="1463" height="352" src="https://private-user-images.githubusercontent.com/16828559/481668584-2bb5a58d-0ecc-471b-8f81-aa4a1380dc48.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYzMDIxNjQsIm5iZiI6MTc1NjMwMTg2NCwicGF0aCI6Ii8xNjgyODU1OS80ODE2Njg1ODQtMmJiNWE1OGQtMGVjYy00NzFiLThmODEtYWE0YTEzODBkYzQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODI3VDEzMzc0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExZmMyMTI0ODU4MGNkMmQzYWNjNTMyYjI2MWQyYWNiMTllNDRmMDMxMmMzMTEyMWMwNmVjOGMzZTY3ZmY2NDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Hu5fybbSeaN23sfFEncQ4xZP5d9YTjHIruiu31p2Pxk">

@chris-campbell Great catch. I added that missing line, and made it content managed. That will require applying another migration (I updated the testing steps to mention this). I also added the missing Simons logo and copywrite line.

chris-campbell reviewed (approved):
This looks good @mattga11ant, lets try it out in QA. Thanks!
